legend-engine-xt-relationalStore-grammar/src/main/java/org/finos/legend/engine/language/pure/compiler/toPureGraph/RelationalCompilerExtension.java

```diff
@@ -483,6 +483,10 @@ public List<Function<Handlers, List<FunctionExpressionBuilderRegistrationInfo>>>
                         ),
                         new FunctionExpressionBuilderRegistrationInfo(Lists.mutable.with(3),
                                 handlers.m(handlers.h("meta::pure::functions::asserts::assertEq_Any_1__Any_1__Function_1__Boolean_1_", false, ps -> handlers.res("Boolean", "one"), ps -> ps.size() == 3 && !handlers.typeOne(ps.get(2), "String")))
+                        ),
+                        // TODO HSO: This is wrong
```

Comment:
this does not seem wrong, the location (relational extension) is what makes it odd to me.